src/Events.php

```diff
@@ -144,6 +145,12 @@ protected function possibly_add_notes() {
 		AddingAndManangingProducts::possibly_add_note();
 		CustomizingProductCatalog::possibly_add_note();
 		GettingStartedInEcommerceWebinar::possibly_add_note();
+		FirstDownlaodableProduct::possibly_add_note();
+
+		if ( $this->is_remote_inbox_notifications_enabled() ) {
+			DataSourcePoller::read_specs_from_data_sources();
+			RemoteInboxNotificationsEngine::run();
+		}
```

Comment:
I'm not sure if we need to add this code, since we already added the remote notifications control [here](https://github.com/woocommerce/woocommerce-admin/blob/adb7165ff45bcab66a2f804eede6cedbd6c0eacd/src/Events.php#L98).

Reply:
You're totally correct. I think it got in while I was rebasing. Thank you for the catch!